### ISSUE TITLE
Selected image delete text instead when hit "Delete" button

### ISSUE THREAD

**Issue by wongmunchoong** [brijeshb42/medium-draft#27]
I find it funny when i selected the image but when i hit delete, it erase the caption instead of whole image block.

However, no issue when hit "Backspace".

thanks.

**Comment by brijeshb42** [comment]
Good catch. I didn't take into account the Delete key (mostly because its not on my keyboard). Will have to fix this in upcoming update.
